documentation/config_file_metrics.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: "Puppet Server Configuration Files: metrics.conf"
+canonical: "/puppetserver/latest/config_file_metrics.html"
+---
+
+The `metrics.conf` file configures Puppet Server's [metrics services](./puppet_server_metrics.markdown) and [v2 metrics API](./metrics-api/v2/metrics_api.markdown).
+
+## Settings
+
+All settings in the file are contained in a HOCON `metrics` section.
+
+-   `server-id`: A unique identifier to be used as part of the namespace for metrics that this server produces.
+
+-   `registries`: A section that contains settings to control which metrics are reported, and how they're reported.
+    -   `puppetserver`: A section that wraps registries settings.
```

Comment:
AFAICT, this actually maps to a registry name. So there's a default `puppetserver` metrics registry that we use and these are its settings. I guess minimum it's `registry's` rather than `registries`. Is my understanding correct, @camlow325 ?